decred/decred/dcr/account.py

```diff
@@ -139,19 +139,28 @@ def __init__(self, count=0, value=0):
         self.value = value
 
 
-def unblob_check(name, version, actual_pushes, expected_pushes):
+def unblob_check(class_name, cur_version, cur_pushes, check_data):
```

Comment:
What does the `cur_` prefix mean?

Reply:
Current. A bit redundant, I guess. Shall I remove it?

Reply:
Yeah. It doesn't seem to add anything in this context.